diff --git a/src/lib/libraries/extensions/index.jsx b/src/lib/libraries/extensions/index.jsx
index ba18b91..e316f38 100644
--- a/src/lib/libraries/extensions/index.jsx
+++ b/src/lib/libraries/extensions/index.jsx
@@ -1,6 +1,9 @@
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
+import sidekickIconURL from './sidekick/sidekick.svg';
+import sidekickInsetIconURL from './sidekick/sidekick-small.svg';
+
 import musicIconURL from './music/music.png';
 import musicInsetIconURL from './music/music-small.svg';
 
@@ -47,6 +50,15 @@ import gdxforConnectionIconURL from './gdxfor/gdxfor-illustration.svg';
 import gdxforConnectionSmallIconURL from './gdxfor/gdxfor-small.svg';
 
 export default [
+    {
+        name: (
+            <FormattedMessage
+                defaultMessage="SIDEKICK"
+                description="Name for the 'SIDEKICK' extension"
+                id="gui.extension.sidekick.name"
+            />
+        ),
+        extensionId: 'sidekick',
+        iconURL: sidekickIconURL,
+        insetIconURL: sidekickInsetIconURL,
+        description: (
+            <FormattedMessage
+                defaultMessage="Blöcke für interaktive SIDEKICK-Projekte."
+                description="Description for the 'SIDEKICK' extension"
+                id="gui.extension.sidekick.description"
+            />
+        ),
+        featured: true,
+        disabled: false
+    },
     {
         name: (
             <FormattedMessage
