diff --git a/src/lib/libraries/extensions/index.jsx b/src/lib/libraries/extensions/index.jsx
index ba18b91..e316f38 100644
--- a/src/lib/libraries/extensions/index.jsx
+++ b/src/lib/libraries/extensions/index.jsx
@@ -1,6 +1,9 @@
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
+import sidekickMQTTIconURL from './sidekickmqtt/sidekick-mqtt.png';
+import sidekickMQTTInsetIconURL from './sidekickmqtt/sidekick-mqtt-small.png';
+
+import sidekickIconURL from './sidekick/sidekick.svg';
+import sidekickInsetIconURL from './sidekick/sidekick-small.svg';
+
 import musicIconURL from './music/music.png';
 import musicInsetIconURL from './music/music-small.svg';
 
@@ -47,6 +50,15 @@ import gdxforConnectionIconURL from './gdxfor/gdxfor-illustration.svg';
 import gdxforConnectionSmallIconURL from './gdxfor/gdxfor-small.svg';
 
 export default [
+    {
+        name: (
+            <FormattedMessage
+                defaultMessage="SIDEKICK MQTT Extension"
+                description="Name for the 'SIDEKICK MQTT' extension"
+                id="gui.extension.sidekickmqtt.name"
+            />
+        ),
+        extensionId: 'sidekickMQTT',
+        iconURL: sidekickMQTTIconURL,
+        insetIconURL: sidekickMQTTInsetIconURL,
+        description: (
+            <FormattedMessage
+                defaultMessage="Custom Scratch MQTT extension."
+                description="Description for the 'SIDEKICK MQTT' extension"
+                id="gui.extension.sidekickmqtt.description"
+            />
+        ),
+        featured: true,
+        disabled: false
+    },
+    {
+        name: (
+            <FormattedMessage
+                defaultMessage="SIDEKICK Extension"
+                description="Name for the 'SIDEKICK' extension"
+                id="gui.extension.sidekick.name"
+            />
+        ),
+        extensionId: 'sidekick',
+        iconURL: sidekickIconURL,
+        insetIconURL: sidekickInsetIconURL,
+        description: (
+            <FormattedMessage
+                defaultMessage="SIDEKICK Scratch extension."
+                description="Description for the 'SIDEKICK' extension"
+                id="gui.extension.sidekick.description"
+            />
+        ),
+        featured: true,
+        disabled: false
+    },
     {
         name: (
             <FormattedMessage
